seqr/views/apis/anvil_workspace_api.py

```diff
@@ -240,3 +268,89 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
     )
 
     safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
+
+
+def generate_AnVIL_variables(project, data_path, sample_type):
+    pipeline_dag = {
+        "active_projects": [project.guid],
+        "es_enable_export": 'true',
+        "vcf_path": data_path,
+        "project_path": '{}v1'.format(_get_loading_project_path(project, sample_type)),
+        "projects_to_run": [project.guid],
+    }
+    return pipeline_dag
+
+def get_variables(variable_key):
+
+    endpoint = 'api/v1/variables/{}'.format(variable_key)
+    airflow_response = make_request(endpoint, method='GET')
+ 
+    val_str = airflow_response['value']
+    val_dict = json.loads(val_str)
+
+    return val_dict
+
+def update_variables(key, val):
+    
+    endpoint = 'api/v1/variables/{}'.format(key)
+  
+    val_str = json.dumps(val)
+    json_data= {
+        "key": key,
+        "value": val_str
+        }
+
+    make_request(endpoint, method='PATCH', json=json_data)
+
+def get_task_ids(dag_id):
+
+    endpoint = 'api/v1/dags/{}/tasks'.format(dag_id)  
+    airflow_response = make_request(endpoint, method='GET')
+
+    tasks = airflow_response['tasks']
+    task_lst = []
+    for task_dict in tasks:
+        task_lst += [task_dict['task_id']]
+    return task_lst
+    
+
+def trigger_dag(dag_id):
+    json_data = {
+        'conf' : {}
+        }
+    endpoint = 'api/v1/dags/{}/dagRuns'.format(dag_id) 
+    make_request(endpoint, method='POST', json=json_data)
+
+
+def make_request(endpoint, method, **kwargs):
+    # Set the default timeout, if missing
+    if 'timeout' not in kwargs:
+        kwargs['timeout'] = 90
+
+    # Obtain an OpenID Connect (OIDC) token from metadata server or using service
+    # account.
+    google_open_id_connect_token = id_token.fetch_id_token(Request(), AIRFLOW_CLIENT_ID)
+    
+    webserver_url = (
```

Comment:
adding strings together works, but is not very pythonic or easy to maintain. You want to use either `.format` or use an f string as follows:
```
webserver_url = f'https://{AIRFLOW_WEBSERVER_ID}.appspot.com/{endpoint}'
```
You might also want to consider hardcoding the api path here, as you currently duplicate it in every call to this helper:
```
webserver_url = f'https://{AIRFLOW_WEBSERVER_ID}.appspot.com/api/v1/{endpoint}'
```